aiida/backends/tests/__init__.py

```diff
@@ -101,6 +101,10 @@
         'engine.work_chain': ['aiida.backends.tests.engine.test_work_chain'],
         'engine.workfunctions': ['aiida.backends.tests.engine.test_workfunctions'],
         'export_and_import': ['aiida.backends.tests.test_export_and_import'],
+        'export_migration.migration': ['aiida-export-migration-tests.tests.test_migration'],
```

Comment:
can you replace `export_migration.migration` with something that makes it more clear what is being tested here (vs the v01_to_v02, ... tests)?

Reply:
It is testing the functions in `aiida.cmdline.utils.migration.__init__.py`, hence it gets the name of the "module" or folder: `migration`. This is the same reasoning for the other names.
However, if you have another name you prefer, which still makes sense in the context of the files and functions that are being tested, it can quickly be changed. I don't have a better name myself though, sorry.

Reply:
ok, fine